src/ui/pages/Menu/components/Settings/components/RecoverySeedPhrase/RecoverySeedPhrase.scss

```diff
@@ -0,0 +1,107 @@
+.recovery-page-container {
+  margin-bottom: 0 !important;
+
+  .user-tips {
+    margin-top: 0.75rem;
+    margin-bottom: 1.75rem;
+
+    p {
+      margin: 0 0 0.5rem;
+    }
+
+    .card-details-info-block-inner {
+      background: rgba(187, 187, 187, 0.5);
+      padding: 1rem 1.25rem;
+      display: flex;
+      flex-direction: row;
+      margin-bottom: 1.5rem;
+
+      .alert-icon {
+        width: 1.5rem;
+        height: 1.5rem;
+
+        ion-icon {
+          color: var(--ion-color-secondary);
+        }
+      }
+    }
+
+    .card-details-info-block-inner {
+      padding: 1.25rem;
+    }
+
+    .tips {
+      margin: 0;
+      padding-left: 1.25rem;
+
+      li:not(:last-child) {
+        margin-bottom: 0.5rem;
+      }
+    }
+
+    @media screen and (min-width: 250px) and (max-width: 370px) {
+      margin-bottom: 1rem;
+
+      .tips {
+        li {
+          font-size: 0.8rem;
+        }
+      }
+    }
+  }
+}
+
+.recovery-seed-phrase-page-footer {
+  margin: 1rem 0;
+}
+
+.confirm-view-seedpharse {
+  &.responsive-modal .responsive-modal-content {
+    .subtitle {
+      margin: 0 0 1.25rem;
+      text-align: center;
+    }
+
+    ion-list {
+      .condition-item {
+        --min-height: 4.75rem;
+        --ion-safe-area-right: 0;
+        --ion-safe-area-left: 0;
```

Comment:
Add `--background-activated: transparent;` to `.condition-item` so we can remove this grey background from the item when it's being selected (nowhere else we have that so just being consistent):

![31072024091406](https://github.com/user-attachments/assets/71d6825f-a3e4-4103-9373-927e420bbdda)